### PR TITLE
broken doc for multi-network VM mesh expansion, this updates it

### PR DESCRIPTION
**Pull request by christian-posta**
Fix the docs around multi-network VM expansion

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/examples/virtual-machines/multi-network/index.md
Comment (howardjohn) [inline content/en/docs/examples/virtual-machines/multi-network/index.md:72]: nit: snake case seems weird here - we probably accept either one but may want to be consistent

File: content/en/docs/examples/virtual-machines/multi-network/index.md
Comment (howardjohn) [inline content/en/docs/examples/virtual-machines/multi-network/index.md:112]: @nmittler I forgot the latest on this discussion. Does this align with the future plans for MC docs?

File: content/en/docs/examples/virtual-machines/multi-network/index.md
Comment (howardjohn) [inline content/en/docs/examples/virtual-machines/multi-network/index.md:143]: future: we should document alternative DNS setups probably. Or keep waiting for DNS interception I guess, may land in 1.8

File: content/en/docs/examples/virtual-machines/multi-network/index.md
Comment (christian-posta) [reply content/en/docs/examples/virtual-machines/multi-network/index.md]: yah agreed. consistency is key. fixed this.

File: content/en/docs/examples/virtual-machines/multi-network/index.md
Comment (christian-posta) [reply content/en/docs/examples/virtual-machines/multi-network/index.md:112]: Let me know what we should do here.

File: content/en/docs/examples/virtual-machines/multi-network/index.md
Comment (teozkr) [inline content/en/docs/examples/virtual-machines/multi-network/index.md:114]: AFAIK, this is already generated by setting `spec.values.global.multiCluster.enabled: true` in the `IstioOperator`, aside from that: 

1. That `Gateway` listens on `15443` rather than `443`
2. That `Gateway` listens on `*.global` rather than `*.local`

File: content/en/docs/examples/virtual-machines/multi-network/index.md
Comment (teozkr) [inline content/en/docs/examples/virtual-machines/multi-network/index.md:74]: Looks like this isn't passed through to the `ConfigMap`. Should be either `{}` or could be deleted, as far as I understand?

File: content/en/docs/examples/virtual-machines/multi-network/index.md
Comment (teozkr) [inline content/en/docs/examples/virtual-machines/multi-network/index.md:136]: Hm, had to change `gateways` to

```
          gateways:
          - ip: 172.18.0.3
            port: 31798
```

because I don't have a `LoadBalancer` running.

But I still only see the backend service's `PodIP` here, rather than the gateway IP/port.

File: content/en/docs/examples/virtual-machines/multi-network/index.md
Comment (christian-posta) [reply content/en/docs/examples/virtual-machines/multi-network/index.md:74]: thanks, will remove

**Comment by frankbu** [comment]
Added cherrypick-1.7 label since this is broken in the current 1.7 docs. We should try to get this merged ASAP.
cc @istio/wg-environments-maintainers

File: content/en/docs/examples/virtual-machines/multi-network/index.md
Comment (nmittler) [reply content/en/docs/examples/virtual-machines/multi-network/index.md:112]: Yes, this is what we're proposing for Istio 1.7: https://github.com/istio/istio/blob/master/samples/cross-network-gateway/cross-network-gateway.yaml

It's the best solution we have for now, unfortunately.

File: content/en/docs/examples/virtual-machines/multi-network/index.md
Comment (nmittler) [reply content/en/docs/examples/virtual-machines/multi-network/index.md:114]: FYI [new multicluster docs for 1.8](https://docs.google.com/document/d/1EZeTaoFb--Rr4EpJsjMBnRtm1VZT14mJpRbPJiJh37M/edit#heading=h.qspvl9cgiq2g) are going away from using `.global` and just sticking with `.cluster.local`

**Comment by frankbu** [comment]
[This whole section](https://deploy-preview-8024--preliminary-istio.netlify.app/latest/docs/examples/virtual-machines/multi-network/#running-services-on-the-added-vm) doesn't seem to fit here. It uses env variables that haven't been set in earlier sections. The step numbering seems to be messed up (1, 2, 3) ... some stuff ... (1, 2, 3 again). The yaml config format is also off. Should this section just be removed?

Also, notice in the index preview that the section nesting is messed up for the whole doc. For example, all of the install steps are nested under `Preparing your environment`.

**Comment by ericvn** [comment]
For additional information @christian-posta, you can select to see the deploy/netlify test Details which will take you to the generated output. For example, https://deploy-preview-8024--preliminary-istio.netlify.app/latest/docs/examples/virtual-machines/multi-network/#running-services-on-the-added-vm will take you to the generated portion of the page where Frank mentioned the weird numbering. Indentation changes are needed. I can help if needed.

**Comment by christian-posta** [comment]
Ah okay, thanks all. Had a long list of things.. will def get to this later tonight or tomorrow. Will make suggested fixes.

**Review by frankbu — APPROVED** [review]

**Pull request merged**
